.github/workflows/nightly-build.yml

```diff
@@ -40,7 +40,6 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Sign executable and create dmg (OSX)
-        if: ${{ matrix.installer && startsWith(matrix.os, 'macos') }}
```

Comment:
won't removal of this line cause attempts of signing non-OSX binaries and therefore fail?

Reply:
This GH should run only on Mac. This is why it can be removed (I believe).